Makefile

```diff
@@ -28,19 +28,45 @@ build-containers: cli-build
 test-containers:
 	DOCKER_USER=${DOCKER_USER} KADALU_VERSION=${KADALU_VERSION} bash build.sh
 
+# TODO: Use helm values rather than passing from CLI
 gen-manifest:
 	@echo "Generating manifest files, run the following commands"
 	@echo
 	@echo "Install Kadalu Operator followed by CSI Nodeplugin"
 	@echo
 	@mkdir -p manifests
-	@IMAGES_HUB=${IMAGES_HUB} DOCKER_USER=${DOCKER_USER} KADALU_VERSION=${KADALU_VERSION} \
-		python3 extras/scripts/gen_manifest.py manifests/kadalu-operator.yaml
+
+	@echo ---------------------------------------------------------------------
+	@helm template --namespace kadalu helm/kadalu \
+		--set operator.enabled=true \
+		--set operator.kubernetesDistro=kubernetes \
+		--set operator.imagesHub=${IMAGES_HUB} \
+		--set operator.dockerUser=${DOCKER_USER} \
+		--set .Chart.version=${KADALU_VERSION} > manifests/kadalu-operator.yaml
+	@helm template --namespace kadalu helm/kadalu \
+		--set csi-nodeplugin.enabled=true \
+		--set csi-nodeplugin.kubernetesDistro=kubernetes \
+		--set csi-nodeplugin.imagesHub=${IMAGES_HUB} \
+		--set csi-nodeplugin.dockerUser=${DOCKER_USER} \
+		--set .Chart.version=${KADALU_VERSION} > manifests/csi-nodeplugin.yaml
```

Comment:
Can you make this as Make subcommand and then call `DISTRO=kubernetes $(MAKE) helm-manifest`

Reply:
```makefile
DISTRO?=kubernetes
filename_suffix=
ifneq ($(DISTRO),kubernetes)
filename_suffix=-$(DISTRO)
endif

helm-manifest:
	@echo ---------------------------------------------------------------------
	@helm template --namespace kadalu helm/kadalu \
		--set operator.enabled=true \
		--set operator.kubernetesDistro=${DISTRO} \
		--set operator.imagesHub=${IMAGES_HUB} \
		--set operator.dockerUser=${DOCKER_USER} \
		--set .Chart.version=${KADALU_VERSION} > manifests/kadalu-operator${filename_suffix}.yaml
	@helm template --namespace kadalu helm/kadalu \
		--set csi-nodeplugin.enabled=true \
		--set csi-nodeplugin.kubernetesDistro=kubernetes \
		--set csi-nodeplugin.imagesHub=${IMAGES_HUB} \
		--set csi-nodeplugin.dockerUser=${DOCKER_USER} \
		--set .Chart.version=${KADALU_VERSION} > manifests/csi-nodeplugin${filename_suffix}.yaml
```